### ISSUE TITLE
Missing receiver-thoughput-bits value

### ISSUE THREAD

**Issue by wfrischh** [perfsonar/pscheduler#1411]
Perfsonar Server: 
perfsonar.ligo-la.caltech.edu
Kernel:
5.14.0-362.12.1.el9
Perfsonar version:
5.0.8-1.el9

We have setup a send/receive throughput test to two servers, perfsonar.ligo-wa.caltech.edu and perfsonar.ligo.caltech.edu. The summary results via the webpage, https://perfsonar.ligo-la.caltech.edu/toolkit/auth/, only reports the send throughput value. 

When I replicate this via cli
```
[root@perfsonar ~]#   pscheduler task throughput -s perfsonar.ligo-la.caltech.edu -d perfsonar.ligo-wa.caltech.edu --reverse
Submitting task...
Task URL:
https://perfsonar.ligo-la.caltech.edu/pscheduler/tasks/d295ab74-572b-4030-9f7f-a33cb9a79301
Running with tool 'iperf3'
Fetching first run...

Next scheduled run:
https://perfsonar.ligo-la.caltech.edu/pscheduler/tasks/d295ab74-572b-4030-9f7f-a33cb9a79301/runs/43ac162a-dc32-4a85-841d-7bd9f7f6193f
Starts 2024-03-07T10:05:49-06:00 (~27 seconds)
Ends   2024-03-07T10:06:08-06:00 (~18 seconds)
Waiting for result...

* Stream ID 5
Interval       Throughput     Retransmits    Current Window 
0.0 - 1.0      271.56 Mbps    0              7.95 MBytes    
1.0 - 2.0      471.86 Mbps    0              7.95 MBytes    
2.0 - 3.0      471.83 Mbps    0              7.95 MBytes    
3.0 - 4.0      471.86 Mbps    0              7.95 MBytes    
4.0 - 5.0      461.40 Mbps    0              7.95 MBytes    
5.0 - 6.0      450.86 Mbps    0              7.95 MBytes    
6.0 - 7.0      471.89 Mbps    0              7.95 MBytes    
7.0 - 8.0      471.86 Mbps    0              7.95 MBytes    
8.0 - 9.0      471.85 Mbps    0              7.95 MBytes    
9.0 - 10.0     471.83 Mbps    0              7.95 MBytes    
10.0 - 10.1    473.28 Mbps    0              7.95 MBytes    

Summary
Interval       Throughput     Retransmits    Receiver Throughput 
0.0 - 10.1     448.84 Mbps    0              0.00bps
```

However, the JSON data is showing receiver-throughput-bits as having a correct value
```
summary	
end	10.066599
start	0
retransmits	0
throughput-bits	448842154.53501225
throughput-bytes	564789248
receiver-throughput-bits	448674897.394234
```
Running the throughput test without `--reverse` flag via cli returns a `Receiver Throughput` value:
```
[root@perfsonar ~]#   pscheduler task throughput -s perfsonar.ligo-la.caltech.edu -d perfsonar.ligo-wa.caltech.edu 
Submitting task...
Task URL:
https://perfsonar.ligo-la.caltech.edu/pscheduler/tasks/455b4b8e-8193-44f6-8126-ae0bbb71fe0c
Running with tool 'iperf3'
Fetching first run...

Next scheduled run:
https://perfsonar.ligo-la.caltech.edu/pscheduler/tasks/455b4b8e-8193-44f6-8126-ae0bbb71fe0c/runs/3f27ca1e-b0bf-4597-afcb-1c87c3f24701
Starts 2024-03-07T10:08:54-06:00 (~6 seconds)
Ends   2024-03-07T10:09:13-06:00 (~18 seconds)
Waiting for result...

* Stream ID 5
Interval       Throughput     Retransmits    Current Window 
0.0 - 1.0      354.38 Mbps    0              6.38 MBytes    
1.0 - 2.0      377.46 Mbps    0              6.38 MBytes    
2.0 - 3.0      377.48 Mbps    0              6.38 MBytes    
3.0 - 4.0      377.49 Mbps    0              6.38 MBytes    
4.0 - 5.0      367.00 Mbps    0              6.38 MBytes    
5.0 - 6.0      377.49 Mbps    0              6.38 MBytes    
6.0 - 7.0      377.49 Mbps    0              6.38 MBytes    
7.0 - 8.0      377.48 Mbps    0              6.38 MBytes    
8.0 - 9.0      377.52 Mbps    0              6.38 MBytes    
9.0 - 10.0     377.47 Mbps    0              6.38 MBytes    

Summary
Interval       Throughput     Retransmits    Receiver Throughput 
0.0 - 10.0     374.13 Mbps    0              360.85 Mbps
```

Configuring `Send Only` and `Receive Only` Throughput tests via the webpage also only reports the outgoing throughput value.

**Comment by wfrischh** [comment]
Pinging for any updates or additional information needed.

**Comment by mfeit-internet2** [comment]
This is being worked on.

## Notes for reference

Because the JSON is generated by the participant, older participants won't do the corrected version and will result in zero, even on updated leads.

This might be overcome by making the `merged-results` method detect and correct for it.

**Comment by mfeit-internet2** [comment]
This has been fixed and will be released with 5.1.0.
